packages/oui-modal/README.md

```diff
@@ -38,6 +63,60 @@ oui-modal is a package which provide styles for the `oui-modal` component.
 </div>
 ```
 
+### Confirmation or question
```

Comment:
You have no exemple for the `info` modal ?

Reply:
Done